### PR TITLE
Unlock hardcoded dldt parameter for redhat

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,14 +83,14 @@ ifeq ($(BASE_OS),redhat)
   BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.4
   DIST_OS=redhat
   DIST_OS_TAG=$(BASE_OS_TAG_REDHAT)
-  DLDT_PACKAGE_URL=https://storage.openvinotoolkit.org/repositories/openvino/packages/2021.4/l_openvino_toolkit_runtime_rhel8_p_2021.4.582.tgz
```
Comment (dtrawins) [inline Makefile:86]: we should use a list to the public package

File: Makefile [hunk 2]
```diff
@@ -83,14 +83,14 @@ ifeq ($(BASE_OS),redhat)
...
+  DLDT_PACKAGE_URL ?= http://s3.toolbox.iotg.sclab.intel.com/ov-packages/l_openvino_toolkit_runtime_rhel8_p_2021.4.671.tgz
 endif
 
 OVMS_CPP_DOCKER_IMAGE ?= openvino/model_server
 OVMS_CPP_IMAGE_TAG ?= latest
 
 PRODUCT_NAME = "OpenVINO Model Server"
-PRODUCT_VERSION ?= "2021.4"
```
Comment (dtrawins) [inline Makefile:93]: invalid s

**Review by dtrawins — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (dkalinowski) [reply Makefile:86]: there is no public package yet

Comment (dkalinowski) [reply Makefile:86]: done

**Review by dtrawins — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

**Pull request merged**